cmd/crio/config.go

```diff
@@ -108,6 +108,9 @@ cgroup_manager = "{{ .CgroupManager }}"
 # hooks_dir_path is the oci hooks directory for automatically executed hooks
 hooks_dir_path = "{{ .HooksDirPath }}"
 
+# secrets_file_path is the secrets mounts file path
+secrets_file_path = "{{ .SecretsFilePath }}"
```

Comment:
it's not really `secrets` anymore, we use it for _secrets_ but all secrets refs should be changed to "default mounts" or something like this

Reply:
okay, so should I rename it to `default-mounts-file-path`?

Reply:
`default-mounts-path` (no need for `file` in there)

Reply:
SGTM